imagetool/src/main/java/com/oracle/weblogic/imagetool/util/DockerBuildCommand.java

```diff
@@ -168,7 +168,7 @@ private Path createFile(Path filePath) {
                     }
                 }
             } catch (IOException e) {
-                e.printStackTrace();
+                logger.fine("Failed to create Docker log file", e);
```

Comment:
Not throwing exception?

Reply:
I wasn't sure what to do here.  I did not change this because it would change the behavior of the tool.  If the file cannot be created, the docker log is skipped.  If I change it to throw the exception, the tool cannot continue if the docker log file cannot be created.  I think it should have been this way (exits on failure) from the beginning.  But, I was only trying to "cleanup" with minimal changes to behavior.